data/hypervisor.args.in

```diff
@@ -18,7 +18,7 @@ virtio-9p-pci,fsdev=workload9p,mount_tag=rootfs
 -fsdev
 local,id=workload9p,path=@WORKLOAD_DIR@,security_model=none
 -smp
-2,sockets=2,cores=1,threads=1
+2,sockets=1,cores=2,threads=1
```

Comment:
Ah - sorry, it's not your branch that needs to be rebased by the wip/hyperstart one. Shouldn't this PR be for merging the shim into `master` now?